### PR TITLE
 [Feature][Connector-V2][Postgres-cdc]Support for Postgres cdc

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
### Purpose of this pull request

Support for Postgres cdc 

### How was this patch tested?

new e2e test

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by jobmission** [comment]
Very nice need this feature.

File: docs/en/connector-v2/source/Postgre-CDC.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Postgre CDC
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Postgre-CDC.md:1]: How to enable postgres cdc?

Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/source/Postgre-CDC.md:1]: done

**Comment by Carl-Zhou-CN** [comment]
@hailin0 @ic4y @EricJoy2048 @Hisoka-X @TyrantLucifer  Please help review, thank you

**Comment by EricJoy2048** [comment]
> @hailin0 @ic4y @EricJoy2048 @Hisoka-X @TyrantLucifer Please help review, thank you

We have found some serious bugs in PG CDC during testing. Please wait for us to fix them before reviewing this PR.

**Comment by Carl-Zhou-CN** [comment]
Wait for compatibility with the following optimizations  #6106  #5784  #6098  #5785

**Comment by hailin0** [comment]
waiting for ci passed

**Review by EricJoy2048 — DISMISSED** [review]

**Review by Hisoka-X — DISMISSED** [review]
please add new connector into `seatunnel-dist/pom.xml`

**Review by hailin0 — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Comment by W-dragan** [comment]
I found that there is currently no CDC that supports PG partition tables. For example, I created a people table,
 CREATE TABLE public.people (
id int4 NOT NULL,
birth_date date NOT NULL,
country_code bpchar(2) NOT NULL,
"name" text NULL,
CONSTRAINT people_pk PRIMARY KEY (id, birth_date)
)

Then create 3 more partitions

--Publicpeople_ Y2000 definition

CREATE TABLE public.people_y2000 PARTITION OF public.people  FOR VALUES FROM ('2000-01-01') TO ('2001-01-01');

--Publicpeople_ Y2000 foreign keys

--Publicpeople_ Y2001 definition

CREATE TABLE public.people_y2001 PARTITION OF public.people  FOR VALUES FROM ('2001-01-01') TO ('2002-01-01');

--Publicpeople_ Y2001 foreign keys



--Publicpeople_ Y2002 definition

CREATE TABLE public.people_y2002 PARTITION OF public.people  FOR VALUES FROM ('2002-01-01') TO ('2003-01-01');

--Publicpeople_ Y2002 foreign keys
When I configure the use of the People table in my source configuration, I cannot query the table because it is a partition type. Even if I modify the code to forcibly query the table, I cannot perform CDC operations on it
@Carl-Zhou-CN @EricJoy2048

**Pull request merged**